MapboxNavigation/RouteMapViewController.swift

```diff
@@ -287,6 +288,31 @@ class RouteMapViewController: UIViewController {
             mapView.removeArrow()
         }
     }
+
+    func updateCameraAltitude(for routeProgress: RouteProgress) {
+        guard mapView.tracksUserCourse else { return } //only adjust when we are actively tracking user course
+        
+        let zoomOutAltitude = NavigationMapView.zoomedOutMotorwayAltitude
+        let defaultAltitude = NavigationMapView.defaultAltitude
+        let isLongRoad = routeProgress.distanceRemaining >= NavigationMapView.longManeuverDistance
+        
+        
+        let currentStepIsMotorway = currentStep.isMotorway
+        let nextStepIsMotorway = upComingStep?.isMotorway ?? false
+        let isExiting = currentStepIsMotorway && !nextStepIsMotorway //are we exiting a motorway?
+        let notOnMotorway = !currentStepIsMotorway && !nextStepIsMotorway //are we not on a motorway?
```

Comment:
In general, it’s less confusing to express Boolean logic in the positive than as double negatives. In this case, if you turn this variable into `isOnOrApproachingMotorway`, you’ll find that you can inline it below:

```swift
if currentStepIsMotorway || nextStepIsMotorway || isExiting
```

which simplifies to:

```swift
if currentStepIsMotorway || nextStepIsMotorway
```

Is that your intended condition for zooming in?